apm/config.go

```diff
@@ -0,0 +1,85 @@
+package apm
+
+type MonitorConfig struct {
+	PopulateEvents bool
+	Commands       []string
+	Databases      []string
+	Collections    []string
+	Namespaces     []Namespace
+}
+
+type Namespace struct {
+	DB         string
+	Collection string
+}
+
+func stringSliceContains(slice []string, item string) bool {
+	if len(slice) == 0 {
```

Comment:
Nit: This is redundant. The range loop will noop if the length of the slice is 0 or if the slice is nil.

Reply:
you know I copypasta'd this from evergreen :)